packages/discovery-provider/ddl/functions/handle_repost.sql

```diff
@@ -91,19 +96,37 @@ begin
     values
       (new.repost_item_id, milestone_name, milestone, new.blocknumber, new.slot, new.created_at)
     on conflict do nothing;
-    insert into notification
-      (user_ids, type, specifier, group_id, blocknumber, timestamp, data)
-      values
-      (
-        ARRAY [owner_user_id],
-        'milestone',
-        owner_user_id,
-        'milestone:' || milestone_name  || ':id:' || new.repost_item_id || ':threshold:' || milestone,
-        new.blocknumber,
-        new.created_at,
-        json_build_object('type', milestone_name, entity_type || '_id', new.repost_item_id, 'threshold', milestone)
-      )
-      on conflict do nothing;
+
+
+    if entity_type = 'track' then
```

Comment:
did we have to break this out because we need extra param for playlist/albums json_build_object, so we couldnt combine track + playlist like we did before?

Reply:
yup exactly